fang/src/asynk/async_queue.rs

```diff
@@ -242,137 +253,210 @@ where
     }
 
     /// Connect to the db if not connected
-    pub async fn connect(&mut self, tls: Tls) -> Result<(), AsyncQueueError> {
-        let manager = PostgresConnectionManager::new_from_stringlike(self.uri.clone(), tls)?;
+    pub async fn connect(&mut self) -> Result<(), AsyncQueueError> {
+        install_default_drivers();
 
-        let pool = Pool::builder()
-            .max_size(self.max_pool_size)
-            .build(manager)
+        let pool: AnyPool = PoolOptions::new()
+            .max_connections(self.max_pool_size)
+            .connect(&self.uri)
             .await?;
 
+        let conn = pool.acquire().await?;
+
+        let backend = conn.backend_name().to_string();
+
+        drop(conn);
```

Comment:
Is this `drop` necessary? It seems to compile and pass tests without it.

Reply:
It is not necessary, I can delete it.